.github/workflows/visual-comparison.yaml

```diff
@@ -0,0 +1,245 @@
+name: Visual Comparison
+
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize, labeled]
+    branches-ignore: ["version-*", "release-*"]
+    paths:
+      - 'src/**'
+      - 'package.json'
+      - 'package-lock.json'
+
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  MENDABLE_API_KEY: ${{ secrets.MENDABLE_API_KEY }}
+  FULLSTORY_ORGID: ${{ secrets.FULLSTORY_ORGID }}
+  ALGOLIA_ADMIN_KEY: ${{ secrets.ALGOLIA_ADMIN_KEY }}
+  ALGOLIA_APP_ID: ${{ secrets.ALGOLIA_APP_ID }}
+  ALGOLIA_SEARCH_KEY: ${{ secrets.ALGOLIA_SEARCH_KEY }}
+  HTML_REPORT_URL_PATH: reports/${{ github.ref_name }}/${{ github.run_id }}/${{ github.run_attempt }}
+
+
+
+concurrency:
+  group: "visual-comparison-${{ github.workflow }}"
+  cancel-in-progress: true
+
+
+jobs:
+  run-ci:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+    if: ${{ !github.event.pull_request.draft && github.actor != 'dependabot[bot]' && github.actor != 'dependabot-preview[bot]' ||  contains(github.event.pull_request.labels.*.name, 'visual-tests') }}
```

Comment:
Perhaps I missed this but where do you detect if changes are made to the `src/**` or `package.json` files? This was mentioned in the confluence page